camligorkem/assignment2/CommonAncestorTest.java

```diff
@@ -0,0 +1,72 @@
+import java.util.Arrays;
+import java.util.Optional;
+import static org.junit.Assert.assertEquals;
+
+public class CommonAncestorTest {
+    BinaryTree<Integer> bt;
+    BinaryTree<Integer> dupTree;
+    BinaryTree<Integer> emptyTree;
+
+    @org.junit.Before
```

Comment:
nit: why not import org.junit.Before and use @Before here? And in other annotations as well

Reply:
done!